gluten-core/src/main/scala/io/glutenproject/utils/FallbackUtil.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.glutenproject.utils
+
+import io.glutenproject.execution.{GlutenColumnarToRowExecBase, GlutenRowToColumnarExec}
+import io.glutenproject.extension.GlutenPlan
+import org.apache.spark.internal.Logging
+import org.apache.spark.sql.execution._
+import org.apache.spark.sql.execution.adaptive.{AdaptiveSparkPlanExec, AdaptiveSparkPlanHelper, QueryStageExec}
+
+/** attention: if AQE is enable,This method will only be executed correctly after the execution plan
+ * is fully determined
+ */
+
+object FallbackUtil extends Logging with AdaptiveSparkPlanHelper {
+
+  def skip(plan: SparkPlan): Boolean = {
+    return plan match {
+      case _: GlutenColumnarToRowExecBase =>
+        true
+      case _: GlutenRowToColumnarExec =>
+        true
+      case _: BaseSubqueryExec =>
```

Comment:
I note we have a plan named `ColumnarSubqueryBroadcastExec` which extending `BaseSubqueryExec`. Should we skip this one?

Reply:
i think wo shoud  skip  ColumnarSubqueryBroadcastExec,  We should pay attention to the subclasses of TransformSupport, because these classes will call doValidate to determine whether to fallback，  ColumnarSubqueryBroadcastExec is not a  subClasses of  TransformSupport

Reply:
@PHILO-HE